src/basic/parse-util.c

```diff
@@ -589,3 +589,20 @@ int parse_ip_port(const char *s, uint16_t *ret) {
 
         return 0;
 }
+
+int parse_dev(const char *s, dev_t *ret) {
+        int r;
+
+#if SIZEOF_DEV_T == 8
+        uint64_t l;
+        r = safe_atou64(s, &l);
+#elif SIZEOF_DEV_T == 4
+        uint32_t l;
+        r = safe_atou32(s, &l);
+#endif
```

Comment:
i am pretty sure we should serialize and deserializue dev_t as major:minor, as dev_t isn't well-defined, different on all archs. I mean, it might happen that the reader is i386 but the parse is x86-64 even on the same machine, and dev_t differs, hence it's always better to do this properly I think.